server/middleware/setUpWebSecurity.ts

```diff
@@ -15,27 +15,34 @@ export default function setUpWebSecurity(): Router {
     res.locals.cspNonce = crypto.randomBytes(16).toString('hex')
     next()
   })
+
+  const scriptSrc = [
+    "'self'",
+    '*.google-analytics.com',
+    '*.googletagmanager.com',
+    (_req: Request, res: Response) => `'nonce-${res.locals.cspNonce}'`,
+  ]
+  const styleSrc = ["'self'", (_req: Request, res: Response) => `'nonce-${res.locals.cspNonce}'`]
+  const imgSrc = ["'self'", 'data:', '*.google-analytics.com', '*.googletagmanager.com']
+  const fontSrc = ["'self'"]
```

Comment:
you've removed some directives that are needed from each of these, the hot jar ones and unsafe inlines